lib/routes/password.js

```diff
@@ -244,7 +244,8 @@ module.exports = function (
         function notifyAccount() {
           if (devicesToNotify) {
             // Notify the devices that the account has changed.
-            push.notifyPasswordChanged(passwordChangeToken.uid, devicesToNotify)
+            var uid = Buffer(passwordChangeToken.uid, 'hex')
```

Comment:
Ditto here re: buffer vs hex; some code above this calls `db.devices(passwordChangeToken.uid)` which makes me think that `passwordChangeToken.uid` must already be a buffer.